bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java

```diff
@@ -49,8 +49,8 @@ private void showHelp(PrintWriter reply) {
 
     @Override
     public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInstance, Path scratchPath, CommandInvocation command, List<Comment> allComments, PrintWriter reply) {
-        if (!censusInstance.isReviewer(command.user())) {
-            reply.println("Only [Reviewers](https://openjdk.java.net/bylaws#reviewer) are allowed to change the number of required reviewers.");
+        if (!pr.author().equals(command.user()) && !censusInstance.isReviewer(command.user())) {
```

Comment:
This would suggest that my assumption is wrong, and that an author can reduce the number of reviewers required even if they are not a Reviewer. I think this might not be what we want.

Reply:
I think it is fine to let them increase it.

Reply:
Agreed. It's the case of an author reducing it after a Reviewer has increased it that I think should be disallowed.

Reply:
@kevinrushforth and @prrace - I agree, see changes in commit 347594f